config/jobs/kubernetes/wg-k8s-infra/canaries/sig-scalability.yaml

```diff
@@ -172,7 +172,7 @@ periodics:
       args:
       - --cluster=kubemark-500
       - --extract=ci/latest
-      - --gcp-master-size=n1-standard-4
+      - --gcp-master-size=e2-standard-4
```

Comment:
Is 'n1 -> e2' migration really needed?

Currently kubemark-500 runs on n1-standard-4 masters.

Reply:
I would like to test the e2 instance type. I don't think it will introduce regression. We can keep n1 for the migration but ultimately we want to unify the instance type for all the jobs.